scratchapi2/user.py

```diff
@@ -1,47 +1,53 @@
-#pylint: disable=trailing-whitespace
 """
 Scratch API User
 
 Anything returned from API does not contain any attributes.
 Use info() to set/get it.
 
+Contains classes:
+* Project
+* User
+* Classroom
 """
-#pylint: enable=trailing-whitespace
 
 import requests
 from .excs import ScratchAPIError
 from .gclass import GenericData
 
+def _request(path, *opts, api_url="https://api.scratch.mit.edu/"):
+    result = requests.get(api_url + path.format(*opts)).json()
+    if 'code' in result:
+        raise ScratchAPIError(result['code'] + ': ' + result['message'])
+
 class Project(object):
-    """ For Scratch Project """
+    """Represents a Scratch Project."""
+
+    projectid = None

```

Comment:
This is the identifying attribute, so it's helpful to have it set even when it's not known.